.github/workflows/python-package.yml

```diff
@@ -182,10 +405,10 @@ jobs:
         ls -lR output/
         cp output/${{ matrix.os }}/${{ matrix.arch }}/* bitsandbytes/
       # Set up the Python version needed
-    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Why hard code the Python version? Even if we split it into a build and test stage, I think defining the version in one place is better